### PR TITLE
Fix ExternalSize adding ?

### PR DESCRIPTION
**Pull request by ricaun**
This pr is related to #3 Issue.

### CODE REVIEW

**Comment by max-eroshkin** [comment]
yep

File: source/InnoSetup.ScriptBuilder/Directory.Build.Props [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
-    <PropertyGroup>
-        <Version>0.1.2</Version>
```
Comment (max-eroshkin) [inline source/InnoSetup.ScriptBuilder/Directory.Build.Props:3]: Keep the package version untouched. It'll be updated in release

Comment (ricaun) [reply source/InnoSetup.ScriptBuilder/Directory.Build.Props:3]: Done

**Review by ricaun — COMMENTED** [review]
Roolback Version

**Pull request merged**